star_tides/exceptions.py

```diff
@@ -45,28 +30,30 @@ def __init__(
         self,
         error_class: str,
         response_msg: str,
-        logging_msg: str,
         severity: str,
-        http_code: int = 400
+        logging_msg: str = None,
+        http_code: int = HTTPStatus.OK
```

Comment:
This doesn't make sense as a default. Use 500 `HTTPStatus.INTERNAL_SERVER_ERROR` as the default.

Reply:
Not really imo. We only use it in one place.